src/utrfx/util.py

```diff
@@ -8,14 +8,17 @@
 
 def download_fasta_from_ensembl(transcript_id: str, timeout: float = 30.,) -> str:
     """
-    Download a FASTA file containing the cDNA sequence for a given transcript from Ensembl's REST API and
+    Download a FASTA file containing the cDNA sequence (spliced mRNA) for a given transcript from Ensembl's REST API and
     return only the nucleotide sequence (without the FASTA header).
 
+    If the transcript is in the negative strand, it will be given the reverse complement of the spliced mRNA found in Ensembl's REST API,
+    which is it located in the positive strand.         
```

Comment:
The paragraph regarding reverse-complementing should be removed as cDNA is on the correct strand (or strandless, depending on how you approach it) by definition.
Sorry for the fuss!